### DISCUSSION TITLE
Itemize possible destination types

### DISCUSSION THREAD

**Discussion by mygordienko** [eclipse-edc/Connector#737] [Ideas]
In DemoUiApiController I noticed the _dataspaceconnector:s3_ constant being used to configure destination type. 
Shouldn't there be a complete list of such constants?

**Comment by jimmarino** [comment]
Hi,

This is something that needs to be formalized in the runtime. Since the possible supported destination types are determined by the set of configured extensions in a runtime, I propose creating a destination type registry where data transfer extensions can register the destination type they support during bootstrap. This will build the list of supported destination types that can later be queried or used by upstream runtime extensions such as IDS.

This is in our plan as part of the work needed for data transfer and IDS support, so I'll close this issue out.
